management-app/backend/app_management/app.py

```diff
@@ -0,0 +1,71 @@
+from typing import List
+from pydantic import BaseModel
+from pathlib import Path
+
+from kubernetes import client
+from app_management.app_volume import AppVolume
+from app_management.app_container import AppContainer
+
+
+class App(BaseModel):
```

Comment:
You should also see this class: `App`. This is what `AppManager` consumes to deploy a new app.